x/records/keeper/callback_transfer.go

```diff
@@ -33,6 +33,10 @@ func (k Keeper) UnmarshalTransferCallbackArgs(ctx sdk.Context, delegateCallback
 
 func TransferCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, ack *channeltypes.Acknowledgement, args []byte) error {
 	k.Logger(ctx).Info("TransferCallback executing", "packet", packet)
+	if ack.GetError() != "" {
```

Comment:
Do you know for sure that an ack error will go down this branch in practice? I assume so but just double checking since the unit test has a mocked response